spring-boot-rest-2/src/test/java/com/baeldung/unsupportedmediatype/ApplicationUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.unsupportedmediatype;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(UserController.class)
+public class ApplicationUnitTest {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void JsonTestCase() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.post("/user/")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content(  "{\n"
+                                 + "    \"name\": \"Andy\",\n"
+                                 + "    \"age\": 1,\n"
+                                 + "    \"address\": \"Hello world\"\n"
+                                 + "}"))
```

Comment:
These four lines should begin two spaces to the right of the opening quote of "{\n"